src/workbooks/optimization/workbook.json

```
@@ -1570,13 +1783,13 @@
                           }
                         },
                         "customWidth": "50",
-                        "name": "Summary of Windows VMs with or without AHUB"
+                        "name": "Summary of Windows VMs with or without AHUB per Subscription"
```

It's "AHB", not "AHUB". It was renamed a few years ago.
```suggestion
                        "name": "Summary of Windows VMs with or without AHB per subscription"
```